web/sites/default/services.yml

```diff
@@ -55,7 +55,7 @@ parameters:
     #
     # Not recommended in production environments
     # @default false
-    debug: false
+    debug: true
```

Comment:
Similar to the settings.php change, you shouldn't need to update services.yml very often. This value should be set in your settings.local.php to use the development.services.yml file for your local environment. This ensures that we don't push a debugging setting all the way to prod.